packages/kbn-scout/src/playwright/page_objects/maps_page.ts

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the "Elastic License
+ * 2.0", the "GNU Affero General Public License v3.0 only", and the "Server Side
+ * Public License v 1"; you may not use this file except in compliance with, at
+ * your election, the "Elastic License 2.0", the "GNU Affero General Public
+ * License v3.0 only", or the "Server Side Public License, v 1".
+ */
+
+import { ScoutPage } from '..';
+
+export class MapsPage {
+  constructor(private readonly page: ScoutPage) {}
+
+  async gotoNewMap() {
+    // see: https://github.com/elastic/kibana/pull/204607/files#r1932558888
```

Comment:
One more nit, why keep this comment? The method is very straightforward and I don't think we need to link the code back to this discussion. Its just more noise in the code base that does not provide value.

Reply:
To me, it's good to keep it as this "feels" like it's going to come up again in the future.  But no worries, I'll remove.